main.js

```diff
@@ -1,54 +1,83 @@
+        this.index = index;
+    }
+}
 
-// fetching the items from local storage
-const storedBooksList = JSON.parse(localStorage.getItem('books'));
-console.log(storedBooksList.length);
+class Books {
+    constructor() {
+        this.list = [];
+    }
 
-// printing the local storage
-/* eslint-disable */
-for (let i = 0; i < storedBooksList.length; i++) {
-    /* eslint-enable */
-  console.log(storedBooksList[i].title, storedBooksList[i].author);
-  const book = document.createElement('ul');
-  book.className = 'book-menu';
-  book.innerHTML = `<li class="book-name list-group-item">${storedBooksList[i].title}</li>
+    add(title, author) {
+        if (Array.isArray(this.list)) {
+            this.list.push(new Book(title, author));
+        }
+    }
+
+    removeItem(index) {
+        if (Array.isArray(this.list)) {
+            const array = this.list;
+            array.splice(index, 1);
+        }
+    }
+}
+
+// Printing the local storage data
+if (localStorage.getItem('books') != null) {
+    for (let i = 0; i < storedBooksList.length; i++) {
+        const book = document.createElement('ul');
+        book.className = 'book-menu';
+        book.innerHTML = `<li class="book-name list-group-item">${storedBooksList[i].title}</li>
                         <li class="author-name list-group-item">${storedBooksList[i].author}</li>
+                        <li class="index author-name list-group-item">${i}</li>
                         <button type="button" class="remove btn btn-outline-secondary">Remove</button>`;
-  document.querySelector('.book-details').appendChild(book);
+        document.querySelector('.book-details').appendChild(book);
+    }
 }
-/* eslint-enable */
-// Clicking Add button pushing the new entry inside the list
-const list = JSON.parse(localStorage.getItem('books'));
-document.querySelector('#add').addEventListener('click', () => {
-  // pushing the form values in an array
-  list.push({
-    title: document.getElementById('title').value,
-    author: document.getElementById('author').value,
-  });
-
-  // printing the book items
-  const book = document.createElement('ul');
-  book.className = 'book-menu';
-  book.innerHTML = `<li class="book-name list-group-item">${document.getElementById('title').value}</li>
-                        <li class="author-name list-group-item">${document.getElementById('author').value}</li>
-                        <button type="button" class="remove btn btn-outline-secondary">Remove</button>`;
-  document.querySelector('.book-details').appendChild(book);
 
-  // storing the array inside the local storage
-  localStorage.setItem('books', JSON.stringify(list));
+// DELETE THE CURRENT ROW OF THE DATDABASE.
+function deleteRow() {
+    const index = this.previousElementSibling.innerHTML;
+    const books = new Books();
+    const booksStored = JSON.parse(localStorage.getItem('books'));
+    books.list = booksStored;
+    books.removeItem(index);
+    // console.log(books.list);
```

Comment:
Please remove this line of commented code to keep a cleaner codebase.

Reply:
removes the same.